src/predefinedComponents/AvatarHeader/AvatarHeader.js

```diff
@@ -67,7 +68,7 @@ class AvatarHeader extends React.Component {
       extrapolate: 'clamp'
     })
 
-    return (
+    const renderAvatarHeader = () => (
```

Comment:
Why not make it a separate function instead of creating an instance of this function in every renderAvatarHeader?